.github/FUNDING.yml

```
@@ -0,0 +1,12 @@
+# These are supported funding model platforms
+
+github: # [jeremyknight-me]
+patreon: # Replace with a single Patreon username
+open_collective: # Replace with a single Open Collective username
+ko_fi: jeremyknight
+tidelift: # Replace with a single Tidelift platform-name/package-name e.g., npm/babel
+community_bridge: # Replace with a single Community Bridge project-name e.g., cloud-foundry
+liberapay: # Replace with a single Liberapay username
+issuehunt: # Replace with a single IssueHunt username
+otechie: # Replace with a single Otechie username
+custom: # ['https://www.buymeacoffee.com/jeremyknight']
```

YAMLlint (1.37.1)

[warning] too many spaces after colon

(colons)